code/modules/modular_computers/laptop_vendor.dm

```diff
@@ -270,6 +270,9 @@
 
 
 obj/machinery/lapvend/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(user.incapacitated())
+		user << SPAN_WARNING("You can't do that right now!")
+		return
```

Comment:
`attackby` not called for incapacitated mob but machine can be broken, etc (`src.stat != 0`)

Reply:
fixed